recipes/openjdk/all/conanfile.py

```diff
@@ -15,39 +16,49 @@ class OpenJDK(ConanFile):
     settings = "os", "arch"
     no_copy_source = True
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
     def configure(self):
```

Comment:
```suggestion
    def validate(self):
```

Reply:
Unfortunately we can't use `self.info.settings` here 😩